tests/test_target.py

```diff
@@ -0,0 +1,32 @@
+from lib import target
+
+
+class TestTarget(object):
+    def test_URLtarget(self):
+        fqdnurl = "https://www.mozilla.org"
```

Comment:
Should we be validating URLs or hostnames?   I feel like given the context for this job, we'd want hostname and not URL, and let the scan make a determination on what schemes are available

Reply:
IMO, a URL submission should be invalid.